components/datadog/agent/host_linuxos.go

```diff
@@ -50,6 +50,10 @@ func (am *agentLinuxManager) getInstallCommand(version agentparams.PackageVersio
 		commandLine += "REPO_URL=datad0g.com DD_AGENT_DIST_CHANNEL=beta "
 	}
 
+	if version.NightlyChannel {
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
We might refactor this to support `channel` of type `Channel` that is `string` with possible values `prod|beta|nightly`